internal/blueprint/customizations_test.go

```diff
@@ -0,0 +1,158 @@
+package blueprint
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCustomizationsHostname(t *testing.T) {
+
+	var myHostname = "MyHostname"
+
+	TestCustomizations := Customizations{
+		Hostname: &myHostname,
+	}
+
+	bpOrig := Blueprint{
+		Name:           "customizations-test",
+		Description:    "Testing customizations functions",
+		Version:        "0.0.1",
+		Customizations: &TestCustomizations,
+	}
+
+	assert.Equalf(t, &myHostname, TestCustomizations.GetHostname(), "The hostname is different")
+	assert.Equalf(t, &myHostname, bpOrig.Customizations.GetHostname(), "The hostname is different")
+
+}
+
+func TestKernelCustomization(t *testing.T) {
+
+	myKernel := KernelCustomization{
```

Comment:
Same comment for variable name here (and everywhere else). It would be much better to prefix variable names with `expected`.